Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:lts-alpine as builder
+
+COPY . /src
+WORKDIR /src
+
+RUN set -ex \
```

Comment:
This could be nicer with a heredoc https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/

Reply:
That's a buildkit feature and there are still lots of Docker users out there not using buildkit for some (various) reason.